.github/workflows/lint-pr.yml

```diff
@@ -0,0 +1,17 @@
+name: "Lint PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
```

Comment:
Please use full commit hashes when using third-party GH Actions:
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions